src/RefractSourceMap.h

```diff
@@ -37,7 +37,7 @@ namespace drafter {
     }
 
     template<typename T>
-    refract::IElement* LiteralToRefract(const NodeInfo<std::string>& literal)
+    refract::IElement* LiteralToRefract(const NodeInfo<std::string>& literal, ConversionContext& context)
```

Comment:
It is called with the parameter because of the [`ContextBinder`](https://github.com/apiaryio/drafter/pull/295/files/ece2ef58b8aea35970247e09490a91aa7c3e7b84#diff-2ab1b5da6ee8cb5b29287e4ebfd30e94R43) being used to optimise the code in `NodeInfoToElements`.

Maybe we can overload `NodeInfoToElements` with another function definition without the `context` and thus without the `ContextBinder`